diff --git a/pyproject.toml b/pyproject.toml
index 767c9e8..c987bbe 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,18 +25,14 @@
 aiohttp = ">=3.9.2" # Version dependency due to https://github.com/InjectiveLabs/sdk-python/security/dependabot/18
 bech32 = "*"
 bip32 = "*"
-coincurve = "*"
 ecdsa = "*"
 eip712 = "*"
+eth-account = "*"
 grpcio = "*"
-grpcio-tools = "*"
-hdwallets = "*"
+hexbytes = "*"
 mnemonic = "*"
-protobuf = "*"
 requests = "*"
 safe-pysha3 = "*"
-urllib3 = "*"
-websockets = "*"
 web3 = "^6.0"
 
 [tool.poetry.group.test.dependencies]
